fili-core/src/test/groovy/com/yahoo/bard/webservice/web/filters/RoleBasedAuthFilterSpec.groovy

```diff
@@ -15,49 +15,55 @@ import javax.ws.rs.core.SecurityContext
 @Timeout(30)
 class RoleBasedAuthFilterSpec extends Specification {
 
-    JerseyTestBinder jtb
+    private final SystemConfig SYSTEM_CONFIG = SystemConfigProvider.getInstance()
 
-    Class<?>[] getResourceClasses() {
-        [RoleBasedAuthFilterSpec.class, DataServlet.class]
-    }
+    ContainerRequestContext containerRequestContext;
+    SecurityContext securityContext;
+    RoleBasedAuthFilter roleBasedAuthFilter;
 
     def setup() {
-        // Create the test web container to test the resources
-        jtb = new JerseyTestBinder(getResourceClasses())
-    }
+        containerRequestContext = Mock(ContainerRequestContext)
+        securityContext = Mock(SecurityContext)
+        containerRequestContext.getSecurityContext() >> securityContext
 
-    def cleanup() {
-        // Release the test web container
-        jtb.tearDown()
+        roleBasedAuthFilter = new RoleBasedAuthFilter()
+        SYSTEM_CONFIG.setProperty(SYSTEM_CONFIG.getPackageVariableName("user_roles"), "foo,bar");
```

Comment:
We need to make sure this gets cleaned up and re-set to it's original value. Property leaks are SUCH a pain to find.